README.md

```diff
@@ -1,38 +1,81 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# gits-frontend
+
+Local dev environment setup for the frontend of the GITS project. Should not be used in production environments.
 
 ## Getting Started
 
-First, run the development server:
+This will contain a description on how get started to work as a frontend developer.
+
+### Prerequisites
+
+- [Node.js](https://nodejs.org/en/) (v16.14 or higher)
+  We use pnpm as a package manager, but npm and yarn should work as well. If you want to use pnpm, you can install it with `npm install -g pnpm`. npm is installed with Node.js, and yarn can be installed with `npm install -g yarn`.
+- [TypeScript](https://www.typescriptlang.org/) (v5.0.4)
+  The project uses Typescript to add typed programming to the frontend. TypeScript is installed with the project, so you don't need to install it yourself.
+- [Next.js](https://nextjs.org/) (v13.4.7 or higher)
+  Next.js is a React framework that we use to build the frontend. It is installed with the project, so you don't need to install it yourself. We implement some of the features and use the structure of v14 of Next.js. Please refer to the [Next.js documentation](https://nextjs.org/docs) for more information.
+- [React](https://reactjs.org/) (v17.0.2 or higher)
+  React is a JavaScript library for building user interfaces. It is installed with the project, so you don't need to install it yourself. Please refer to the [React documentation](https://reactjs.org/docs/getting-started.html) for more information.
+
+Before you run the project you need to install the dependencies and create the graphql files. You can do this by running the following commands:
 
 ```bash
-npm run dev
+npm run install
 # or
-yarn dev
+yarn install
 # or
-pnpm dev
+pnpm install
 ```
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+This will install all the dependencies for the project. After this you need to create the graphql files. You can do this by running the following command:
 
-You can start editing the page by modifying `pages/index.tsx`. The page auto-updates as you edit the file.
+```bash
+npm run relay
+# or
+yarn relay
+# or
+pnpm relay
+```
 
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.ts`.
+This will create the graphql files for all queries and mutations. You need to run this command every time you change a graphql query or mutation.
+Should the graphql schema change, you need to run the following command:
```

Comment:
we should maybe mention that its done automatically when using pnpm dev

Reply:
Really? At least when I try "pnpm dev" without doing "pnpm relay" beforehand it doesn't work, if some query or mutation changed.